deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Net461.GAC.wxs

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
+     xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <?include $(sys.CURRENTDIR)\Config.wxi?>
+  <Fragment>
+    <ComponentGroup Id="Files.Managed.Net461.GAC" Directory="net461.GAC">
+      <Component Win64="$(var.Win64)">
+        <File Id="net461_GAC_Sigil.dll"
+              Source="$(var.ManagedDllPath)\net461\Sigil.dll"
+              KeyPath="yes" Checksum="yes" Assembly=".net"/>
+      </Component>
+    </ComponentGroup>
+  </Fragment>
+</Wix>
```

Comment:
Can this be managed in `SyncMsiContent`?
The `Files.Managed.Net45.GAC.wxs` is managed through there, though it is a little different as this is for a single dll.

Reply:
Would be great if we could automate it, but not sure how that would work. `Sigil` is a special case.

Reply:
On second though, until we have more special cases like `Sigil`, there's isn't really anything to automate.